src/domain/organization/organizationForm/__tests__/OrganizationForm.test.tsx

```diff
@@ -78,3 +79,27 @@ test('should add and remove merchant', async () => {
     expectedInputLabel: 'Paytrail-kauppiastunnus *',
   });
 });
+
+test('superuser can edit users', async () => {
+  renderComponent();
+  await loadingSpinnerIsNotInDocument();
+
+  expect(
+    screen.getByRole('combobox', { name: 'Pääkäyttäjät' })
+  ).not.toBeDisabled();
+});
+
+test('normal user cant edit users', async () => {
+  const adminMocks = [
+    mockedOrganizationsResponse,
+    mockedOrganizationClassResponse,
+    mockedOrganizationClassesResponse,
+    mockedUserResponse,
+    mockedUsersResponse,
+  ];
+
+  render(<OrganizationForm />, { mocks: adminMocks });
+  await loadingSpinnerIsNotInDocument();
+
+  expect(screen.getByRole('combobox', { name: 'Pääkäyttäjät' })).toBeDisabled();
```

Comment:
Seems like only the organization admin field is tested to be disabled. Maybe test the same for the rest of the user fields too (Ilmoittautumisen pääkäyttäjät, Taloushallinnon pääkäyttäjät, Peruskäyttäjät)?

In the superuser test, it could be additionally tested that the same fields are not disabled.

Reply:
Added those cases.